includes/functions.php

```diff
@@ -28,11 +28,13 @@ function network_exists( $network_id ) {
  *
  * @return array Networks available on the installation
  */
-function get_networks() {
-	global $wpdb;
+ if ( ! function_exists( 'get_networks') ){
+	function get_networks() {
```

Comment:
get_networks in core accepts  an array of args. I believe better compablity would be 
`function get_networks( $args = '' ) {`
In this plugin it would do nothing....